src/model/database/db_build.sql

```diff
@@ -0,0 +1,49 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users,articles,contacts,chats CASCADE;
+
+CREATE TABLE users (
+  user_id SERIAL PRIMARY KEY ,
+	user_name VARCHAR(100) NOT NULL,
+	user_email TEXT NOT NULL UNIQUE,
+	user_password TEXT,
+	permission  VARCHAR(100) DEFAULT 'user'
+);
+INSERT INTO users (user_name,user_email,user_password,permission) VALUES
+('salwa','salwa@hotmail.com','123','admin');
+
+INSERT INTO users (user_name,user_email,user_password,permission) VALUES
+('ons','ons@hotmail.com','1861997','doctor'),
+('kannan','kannan@hotmail.com','rami','doctor'),
+('mohanned','mohanned@hotmail.com','fofo','doctor');
+
+CREATE TABLE articles (
+article_id SERIAL PRIMARY KEY ,
+article_title VARCHAR(250) NOT NULL,
+article_img TEXT ,
+article_body TEXT NOT NULL
+);
+
+INSERT INTO articles  (article_title,article_img,article_body) VALUES
```

Comment:
only suggestion: even if it just a demo it would be good if you add a real data 
off course you don't have to change it